CMakeLists.txt

```diff
@@ -31,9 +33,6 @@ if(NOT AOTRITON_NO_PYTHON)
   add_subdirectory(third_party/pybind11)
 endif()
 
-# Must be after pybind11
-set(CMAKE_CXX_COMPILER hipcc)
```

Comment:
We do not need to use `hipcc` since we are not compiling any kernels, so a host compiler should be good enough.